lib/soap.ex

```diff
@@ -1,12 +1,33 @@
 defmodule Soap do
   @moduledoc """
-  Provides a functions for send SOAP requests.
+  The SOAP client for Elixir based on HTTPoison (for send requests) and SweetXml (for xml parsing).
```

Comment:
I think there would be better to talk about the modules of the library and show the flow of use.
e.g. https://github.com/elixir-ecto/ecto/blob/master/lib/ecto.ex